### ISSUE TITLE
Android examples with cmake 

### ISSUE THREAD

**Issue by soham24** [halide/Halide#4095]
android examples are now obsolete. they use android21 platform and old platform tools and sdk tools to build. Its high time that they should get updated with the cmake build system.

**Comment by soham24** [comment]
I am trying hard to run HelloAndroidCamera2 example on linux, still not succeed

**Comment by steven-johnson** [comment]
We would be very very happy for someone to upgrade our Android examples to use CMake, a PR would be much welcomed.

**Comment by soham24** [comment]
@steven-johnson  this guy has posted one example 
@PK15946
https://github.com/PK15946/Halide-Android-example

**Comment by steven-johnson** [comment]
Thanks for the pointer. I am unlikely to have bandwidth to work on this anytime soon -- if anyone else would like to adapt this into a proper PR then we would welcome it.

**Comment by soham24** [comment]
@steven-johnson makes sense. I will try to do it after my current project work gets done.

**Comment by abadams** [comment]
> [!NOTE]
> **Automated triage:** this issue was flagged by Claude as likely safe to close, for the
> reason below. The reasoning was checked against the current source, and a second pass
> tried and failed to refute it, but no human verified this specific text.
> **If the reason is wrong, please reopen the issue and say so** — some fraction of these
> are expected to be wrong.

**Reason (CLOSE_DUPLICATE):** Asks for the Android sample apps to be ported to the CMake build. That work is still undone, but it is already tracked (with the two apps named explicitly) by the newer umbrella issue #5374, which the source itself points at.

**Evidence:** apps/CMakeLists.txt:58-59: `# add_app(HelloAndroid)  # TODO(#5374): missing CMake build` / `# add_app(HelloAndroidCamera2)  # TODO(#5374): missing CMake build`; apps/HelloAndroid still builds via jni/Android.mk and build.gradle pinned to gradle plugin 1.2.2

Closing as a duplicate of #5374 ("Add CMake builds to all apps"), which explicitly tracks HelloAndroid and HelloAndroidCamera2 — see the TODO(#5374) markers at apps/CMakeLists.txt:58-59. The work is still wanted; please follow it there.
